plugin.js

```diff
@@ -0,0 +1,21 @@
+const fp = require('fastify-plugin')
+const jitson = require('jitson')
+
+function plugin (fastify, options, next) {
+  fastify.addContentTypeParser('application/json', { parseAs: 'string' }, function (req, body, done) {
+    const parse = jitson(options)
```

Comment:
This line should go outside the `addContentTypeParser` function? In this way, we cache the `parse` object and we can avoid to create it for each json request.
Moreover, if we recreate it each request we are not losing the schema sampling?

I think this part could be optimized with a Map() where the key is the route path and the value the jitson object

WDYT?

Reply:
Yes, good spot!

Reply:
@Eomm You are absolutely right